src/main/java/org/jenkinsci/plugins/workinghours/WorkingHoursQueueTaskDispatcher.java

```diff
@@ -120,7 +121,7 @@ public boolean canRunNow(Actionable itemActionable,
 
         EnforceBuildScheduleAction action = itemActionable.getAction(EnforceBuildScheduleAction.class);
 
-        WorkingHoursConfig config = WorkingHoursConfig.get();
+        WorkingHoursPlugin config = ExtensionList.lookup(WorkingHoursPlugin.class).get(0);
```

Comment:
Just as an FYI, there are several unit tests that will need to be updated to access config with the new pattern. We can do that later when we're sure of this approach though